Assets/Scripts/UI/InGameUI/DevConsole/DevConsole.cs

```diff
@@ -815,7 +818,9 @@ private void LoadCommands()
                 new Command("GetRoomIDs", CoreCommands.GetAllRoomIDs, "Logs all the room IDs"),
                 new Command<int, string, Vector3>("DoBuild", CoreCommands.DoBuild, CoreCommands.DoBuildHelp),
                 new Command("DirtyTileGraph", CoreCommands.InvalidateTileGraph, "Invalidates the tile graph"),
-                new Command("GetCharNames", CoreCommands.GetCharacterNames, "Logs all the character names"));
+                new Command("GetCharNames", CoreCommands.GetCharacterNames, "Logs all the character names"),
+                new Command<Vector2>("NewCharacter", CoreCommands.NewCharacter, "Creates a new character at position provided"),
```

Comment:
Since these are so small, you can just add these changes in your other PR if its easier. if @koosemose really wants the NewCharacterCenter he can just do it himself :)